cp2/ControlPanel/static_cp/js/status.js

```diff
@@ -1,8 +1,7 @@
     //-----------------------------------------Status Module-----------------------------------------//
 
-    function estate(jsonObj, idDiv, moduleName) {
-        var status = jsonObj['status'];
-        if (status === "OK") {
+    function estate(status, idDiv, moduleName) {
```

Comment:
Let's change the name of this function "estate" to "moduleState" or similar.
Otherwise it look like a real estate property :S